### PR TITLE
[8.0] l10n_be_mis_reports

### PR DESCRIPTION
**Pull request by sbidoul**
This is a data module with the belgium P&L and Balance sheet in mis_builder format.
- [x] This PR depends on https://github.com/OCA/account-financial-reporting/pull/86

### CODE REVIEW

File: l10n_be_mis_reports/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+To obtain correct results, the account codes prefixes must match the official
+Belgium chart of account.
+
+Usage
+=====
+
+To use this module, you need to go to 
+Accounting > Reporting > MIS Reports and use the buttons
+available on the previously configured reports such as previous,
```
Comment (alexis-via) [inline l10n_be_mis_reports/README.rst]: Previous -> Preview

Comment (sbidoul) [inline l10n_be_mis_reports/README.rst]: Well spotted, thanks! It is fixed.

**Comment by sbidoul** [comment]
rebased

File: l10n_be_mis_reports/README.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+This modules provides MIS Builder Report templates for the Belgium
+P&L and Balance Sheet according to the official models.
+
+Installation
+============
+
+The normal Odoo module installation procedure applies.
+
+This module depends on the mis_builder module.
```
Comment (adrienpeiffer) [inline l10n_be_mis_reports/README.rst]: It might be interesting to precise from which repository

File: l10n_be_mis_reports/README.rst [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+Belgium chart of account.
+
+Usage
+=====
+
+To use this module, you need to go to 
+Accounting > Reporting > MIS Reports and use the buttons
+available on the previously configured reports such as preview,
+export, add to dashboard.
+
```
Comment (adrienpeiffer) [inline l10n_be_mis_reports/README.rst:41]: Can you add _Try me on Runbot_ link ?

File: l10n_be_mis_reports/README.rst [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+Known issues / Roadmap
+======================
+
+N/A
+
```
Comment (adrienpeiffer) [inline l10n_be_mis_reports/README.rst:50]: Can you add bug tracker section ?

File: l10n_be_mis_reports/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+#     even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+#     PURPOSE.  See the GNU Affero General Public License for more details.
+#
+#     You should have received a copy of the GNU Affero General Public License
+#     along with l10n_be_mis_reports.
+#     If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    'name': "l10n_be_mis_reports",
```
Comment (adrienpeiffer) [inline l10n_be_mis_reports/__openerp__.py]: Use the common name

File: l10n_be_mis_reports/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+    'summary': """
+        MIS Report templates for the Belgium P&L and Balance Sheets""",
+
+    # 'description': put the module description in README.rst
+
+    'author': "ACSONE SA/NV",
```
Comment (adrienpeiffer) [inline l10n_be_mis_reports/__openerp__.py]: Add OCA as author

File: l10n_be_mis_reports/__openerp__.py [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+    'website': "http://acsone.eu",
+
+    # Categories can be used to filter modules in modules listing
+    # Check http://goo.gl/0TfwzD for the full list
+    'category': 'Uncategorized',
```
Comment (adrienpeiffer) [inline l10n_be_mis_reports/__openerp__.py]: Reporting

File: l10n_be_mis_reports/__openerp__.py [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+    'version': '0.1',
```
Comment (adrienpeiffer) [inline l10n_be_mis_reports/__openerp__.py]: Modify version format according with https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers

**Comment by adrienpeiffer** [comment]
It is green :smile:  I also added some small remarks about the readme file and the manifest.

File: l10n_be_mis_reports/__openerp__.py [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+    'depends': [
+        'mis_builder',
+    ],
+
+    # always loaded
+    'data': [
+        'data/mis_report_pl.xml',
+        'data/mis_report_bs.xml',
+    ],
+}
```
Comment (adrienpeiffer) [inline l10n_be_mis_reports/__openerp__.py:42]: Can you remove comments lines ?

**Comment by sbidoul** [comment]
Thanks @adrienpeiffer. I handled your comments.

File: l10n_be_mis_reports/README.rst (hunk 3 above)
Comment (adrienpeiffer) [inline l10n_be_mis_reports/README.rst:41]: Thanks !

File: l10n_be_mis_reports/README.rst (hunk 4 above)
Comment (adrienpeiffer) [inline l10n_be_mis_reports/README.rst:50]: Thanks !

**Comment by adrienpeiffer** [comment]
:+1: Thanks!

File: l10n_be_mis_reports/README.rst [hunk 10]
```diff
@@ -0,0 +1,77 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
```
Comment (lmignon) [inline l10n_be_mis_reports/README.rst:3]: Can you make the module title a real title by adding some "===" above the title (https://github.com/OCA/maintainer-tools/pull/109)?

**Comment by lmignon** [comment]
A little comment but :+1:

**Comment by sbidoul** [comment]
@lmignon done.

**Comment by sbidoul** [comment]
This PR is ready to merge. Can a core maintainer have a last look and do the merge to respect the no-self-merge rule?

**Comment by alexis-via** [comment]
arf, stupid no-self-merging rule...

**Comment by sbidoul** [comment]
@alexis-via I would not say stupid, but this is indeed the situation of a small community that you mentioned on the mailing list. 

That said, I don't think I'll do much harm by taking my responsibility and merging this myself, as long as self-merging remains an exception.

**Comment by pedrobaeza** [comment]
Merging as it has 3 thumbs up

**Pull request merged**